src/main/java/dev/jbang/cli/Run.java

```diff
@@ -194,6 +194,8 @@ String generateCommandLine(Source src, RunContext ctx) throws IOException {
 					}
 				}
 
+				optionalArgs.add("--execution=local");
```

Comment:
what is that flag trying to achieve?

Reply:
like, my jshell has no documentation for that flag afaics but it does not complain about it...

Reply:
jshell, by default, starts up a VM in the background to which it connects. But this means you can't read stdin from that VM because it's not connected to the terminal. This option tells jshell to not do that and just use the local process for the VM, which then inherits the stdin from jshell. It even makes jshell a bit faster it seems.

Reply:
Hmmm...interesting...what's the downside ?

Reply:
Not sure, the information on this is very limited. I'm assuming it's more efficient when you call jshell a lot.

Reply:
i'm hesitant to do this specific change but lets merge in and we can revert if some usecase fails